src/scripts/lexmatch-sssom-compare.py

```diff
@@ -296,6 +296,14 @@ def get_unmapped_df(
     ]
 
     new_df = pd.concat([unmapped_lex_df, unmapped_mondo_df], axis=0)
+    if 'confidence' not in new_df.columns:
+        new_df['confidence'] = 0.5
```

Comment:
Not needed. The[ PR in `sssom-py`](https://github.com/mapping-commons/sssom-py/pull/548) should eliminate the need for this.